docs/source/monitors.rst

```diff
@@ -1,5 +1,3 @@
-.. _monitors:
```

Comment:
Why remove this?

It’s not wrong to remove it if it is not used internally and it is _probably_ not used externally yes (through InterSphinx). However, it’s good to have references so that future projects may use InterSphinx to link to this section without having to rely on the documentation page source file path.

Reply:
When building the docs, sphinx complained that this is a duplicated reference. We have `Monitors` as title some lines below, so both shares the same reference name. So removing this will not affect existing internal or external references.